### PR TITLE
Wagalytics Multisite Support

### PR DESCRIPTION
**Pull request by KalobTaulien**
Added in multisite Wagtail support. Each site can be configured to hold its own GA View ID, and Key Content (or Key Filepath). If there's more than one site _and_ the WAGALYTICS_SETTINGS are set, a site switcher will display (image below)
image

Multisite support requires `wagtail.contrib.settings` to be enabled (not a problem for your website, but still a noteworthy point). 

I've upgraded the `re_path` URLs to use Django `path`s. And also added Django Error Messages to the `dashboard` view to help guide setup.

Continued support for Single Site instances is included. 

Added some basic tests, using PyTest. A proper GA VIEW ID and KEY.json (or content_of_KEY.json) is required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# Wagtail Analytics
+# Wagtail Analytics - NYPR Fork
```
Comment (tomdyson) [inline README.md]: not 'NYPR Fork' here :)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+
+### Includes the following patches on top of upstream master:
```
Comment (tomdyson) [inline README.md]: Not relevant here

File: README.md [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+PR | Status
+-- | ------
+Multisite Analytics Support | Open
+
+(Last Updated 12/17/19 for Wagalytics v2.6.1)
 
 This module provides a simple dashboard of Google Analytics data, integrated into the Wagtail admin UI. Tested on Wagtail 1.4+.
```
Comment (tomdyson) [inline README.md]: Now Wagtail 2.x?

File: README.md [hunk 4]
```diff
@@ -27,6 +34,44 @@ If you get CryptoUnavailableError errors, you probably need to `pip install PyOp
 
 Ensure that your code snippet is included on each page you want to be tracked (likely by putting it in your base.html template.) (Admin > Property > Tracking Code)
 
+## Multisite Support 
+
+To enable multisite support you'll need to update your Wagalytics settings _and_ have `wagtail.contrib.settings` installed. Sites can use a `GA_KEY_FILEPATH` or a `GA_KEY_CONTENT` key, but it's best not to use both. 
+
+In the snippet below, you'll see `site_id`. This is the ID (Priamry Key) of your Wagtail Site.
```
Comment (tomdyson) [inline README.md]: Priamry -> Primary

File: README.md [hunk 5]
```diff
@@ -27,6 +34,44 @@ If you get CryptoUnavailableError errors, you probably need to `pip install PyOp
...
+    },
+    site_id: { 
+        'GA_VIEW_ID': 'ga:xxxxxxxx', 
+        'GA_KEY_CONTENT': 'content_of_your_key.json',
+	}
+}
+```
+For every Wagalytics site you add in your multisite `WAGALYTICS_SETTINGS` you'll need to make sure you have the proper GA View ID and API Key. One View ID and API Key won't work for all your sites automatically. 
+
+Here's a working example of _almost live_ WAGALYTICS_SETTINGS:
```
Comment (tomdyson) [inline README.md]: '_almost live_' ?

**Comment by drimacus182** [comment]
Does this PR eliminate per-page statistics added in 584e8aeb3122e337c1a3f95944aed94492c90c30 commit? (See `insert_editor_js` hook  in  `wagtail/wagtail_hooks.py` )

**Comment by drimacus182** [comment]
It seems nobody is here :-(

**Comment by jacobwegner** [comment]
@drimacus182 There has been some discussion on #23 

Here is a Gist with a working hook:

https://gist.github.com/jacobwegner/011ddeb93c5aeeccc9277eeb6ccaea56

You'd want to copy `wagtail_hooks.py` into your project (or add to an existing `wagtail_hooks.py` script):

https://docs.wagtail.io/en/stable/reference/hooks.html#hooks

**Pull request merged**